layer/event_logging.h

```diff
@@ -0,0 +1,288 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef STADIA_OPEN_SOURCE_PERFORMANCE_LAYERS_EVENT_LOGGING_H_
+#define STADIA_OPEN_SOURCE_PERFORMANCE_LAYERS_EVENT_LOGGING_H_
+
+#include <string>
+#include <vector>
+
+namespace performancelayers {
+// Set of supported value types for an attribute.
+enum ValueType { String, Int64, VectorInt64 };
```

Comment:
Can you also update the enum names to start with the `k` prefix? See https://google.github.io/styleguide/cppguide.html#Enumerator_Names